region/maridia/outer.json

```diff
@@ -6225,6 +6503,89 @@
               "usableComingIn": false,
               "openEnd": 1
             }
+          ],
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": false,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Through Tube (Bottom to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,3],
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "Morph",
+                    {"or":[
+                      "Gravity",
+                      {"and":[
+                        "canSuitlessMaridia",
+                        "canUseFrozenEnemies",
+                        "HiJump",
+                        "h_canCrouchJumpDownGrab"
+                      ]},
+                      {"and":[
+                        "canSuitlessMaridia",
+                        {"ammo": { "type": "Super", "count": 1 }},
+                        "canTrickyUseFrozenEnemies"
+                      ]},
+                      {"and":[
+                        "canSuitlessMaridia",
+                        "canTrickyJump",
+                        "canSpringBallJumpMidAir",
+                        "canTrickyUseFrozenEnemies"
+                      ]}
+                    ]}
+                  ],
+                  "devNote": "There are many ways to get up with Gravity, but it can be done with no other items with a gravity jump."
+                }
+              ]
+            },
+            {
+              "leavesWithArtificialMorph": true,
+              "strats": [
```

Comment:
You can artificial morph into the room from below and IBJ up to either tube entrance?

Reply:
You can, but because of the limitations of how the logic allows multi room strats, these arent possible to represent. It requires starting at the node in question, and then going to the doors below, which requires morph, so no artificial strats can be added currently